src/modules/common/hooks/useCustomQuery.js

```diff
@@ -28,9 +30,17 @@ export const useCustomQuery = ({
   client = defaultClient
 }) => {
   const [{ chainID }] = useCurrentApplication();
+  const network = useSelector(state => state.network);
+  const networkName = getNetworkName(network)
+
+  const axiosConfig = {
+    ...config,
+    params: { ...(config?.params ?? {}), network: networkName }
```

Comment:
I think instead of doing this you can make `config` on line 28 this `config = {}` then here you can do this

params: { ...config.params , network: networkName }